### ISSUE TITLE
numpy 2 compatible release

### ISSUE THREAD

**Issue by tmct** [sktime/pytorch-forecasting#1707]
Hi,

In August numpy 2 support was added in #1599, but this hasn't been released to pypi yet - 1.1.1 came out in September.

Please could a version of pytorch-forecasting be released with that change in?

Thanks,
Tom

**Comment by fnhirwa** [comment]
@fkiraly,  should we make a Pypi release for numpy 2 support?

**Comment by fkiraly** [comment]
yes, we probably should. I repeatedly got stuck with trying to fix the `conda` release, but we can separate that.

@fnhirwa, do you want to prepare the release or should I? I will also ping on discord to check.

**Comment by fnhirwa** [comment]
> yes, we probably should. I repeatedly got stuck with trying to fix the `conda` release, but we can separate that.
> 
> @fnhirwa, do you want to prepare the release or should I? I will also ping on discord to check.

Sure, I can prepare the release with no problem.

**Comment by fkiraly** [comment]
Pinged you on discord to suggest time slots for releasing together with screenshare, to ensure robuts handover of practical knowledge around rhe release.
